cmd/kubeadm/app/cmd/init.go

```diff
@@ -283,7 +284,11 @@ func (i *Init) Run(out io.Writer) error {
 		return err
 	}
 
-	if err := addonsphase.CreateEssentialAddons(i.cfg, client); err != nil {
+	if err := dnsaddonphase.CreateEsentialDNSAddon(i.cfg, client); err != nil {
```

Comment:
Wasn't certain if we wanted to keep this as just `addonsphase`. I can change if so.